src/Compilers/CSharp/Portable/Binder/BinderFlags.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// A specific location for binding.
     /// </summary>
     [Flags]
-    internal enum BinderFlags : uint
+    internal enum BinderFlags : ulong
```

Comment:
>ulong [](start = 32, length = 5)

Is this change needed?

Reply:
Try `1u << 31`

---
In reply to: [195607839](https://github.com/dotnet/roslyn/pull/27807#discussion_r195607839) [](ancestors = 195607839,195579166)